Serverless Deployment not working

Trying to deploy but this error occurred in all tries ...

Command sam failed with exit code 1 signal null
Uploading to 954276bde540ef29a7b5ddabc1f03673  1163 / 1163.0  (100.00%)
Uploading to dbfc6bdb796c6f50de4cc1b223820652.template  10184 / 10184.0  (100.00%)
Error: Failed to create/update the stack: liveeventdeploymentstack-broadcast, Waiter StackCreateComplete failed: Waiter encountered a terminal failure state

,Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
make: *** [build_upload] Error 1

Command node failed with exit code 1 signal null
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
make: *** [build_upload] Error 1

These are installed services at macOS
,aws-cli/2.0.31 Python/3.7.4 Darwin/19.5.0 botocore/2.0.0dev35
,
,SAM CLI, version 0.53.0
,

nidobda@MacBook-Pro serverless % node ./deploy.js -r us-east-1 -b mdliveversion4 -s mdsliveversion4

,6.13.4
,
,aws-cli/2.0.31 Python/3.7.4 Darwin/19.5.0 botocore/2.0.0dev35
,
,SAM CLI, version 0.53.0
,
Installing node dependencies
,audited 1049 packages in 5.548s

43 packages are looking for funding
  run `npm fund` for details

found 21 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
,npm WARN chime-sdk-react-demo@ No repository field.
npm WARN chime-sdk-react-demo@ No license field.


,audited 531 packages in 4.546s

15 packages are looking for funding
  run `npm fund` for details

found 4277 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
,npm WARN live-events-lambdas@ No description
npm WARN live-events-lambdas@ No repository field.
npm WARN live-events-lambdas@ No license field.


,,
Compiling frontend assets
,
> chime-sdk-react-demo@ release /Users/nidobda/Documents/dev/aws/amazon-chime-live-events
> npm run fix-eval && rm -rf dist/* && npm run package:attendee && npm run package:moderator && npm run package:talent && npm run package:broadcast


> chime-sdk-react-demo@ fix-eval /Users/nidobda/Documents/dev/aws/amazon-chime-live-events
> ./node_modules/replace/bin/replace.js -r --include='*.js' 'eval.*\(moduleName\);' 'undefined;' ./node_modules/protobufjs/


> chime-sdk-react-demo@ package:attendee /Users/nidobda/Documents/dev/aws/amazon-chime-live-events
> webpack --config src/apps/attendee/webpack.prod.js

Shared dirname is /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/apps
Template: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/index.html
Prod path: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/dist
Hash: 41ee49ceb2bfee3f063d
Version: webpack 4.42.1
Time: 9525ms
Built at: 07/24/2020 1:33:08 AM
                                 Asset      Size  Chunks                                Chunk Names
  0747be4cba2f8600544f4f09645d971a.eot  33.6 KiB          [emitted]                     
  18df003f90a7f1313b98b23384ec4f1b.svg   876 KiB          [emitted]              [big]  
31015e861aeb30ff8355fab3e876a6bd.woff2  74.8 KiB          [emitted]                     
  32267f3b4db33a7c85ee31d9b317c9e4.ttf   198 KiB          [emitted]                     
  3c45c98f6f6c1b009c082510d0255ef4.svg   699 KiB          [emitted]              [big]  
  5818bbd07dfd7095920a592049f5d9bd.ttf   130 KiB          [emitted]                     
  607a77859353c0cc1882b51e1e269d38.eot   130 KiB          [emitted]                     
  7eefe04dc985dd4ad38bd7f4ec0c20a0.eot   198 KiB          [emitted]                     
 8b892b7904a39e37c53dc5a7534bb60e.woff  16.4 KiB          [emitted]                     
9451d5fee89b51a20f8a44c56a4f6c1a.woff2  77.6 KiB          [emitted]                     
 a940d584750708f5435ce2c523498ddb.woff   101 KiB          [emitted]                     
  ab47faeab8ddc5f809ec2a5377c1fbe4.svg   141 KiB          [emitted]                     
      attendee.af3e7b864eca747edcc0.js  1.98 MiB       0  [emitted] [immutable]  [big]  main
                         attendee.html  1.51 KiB          [emitted]                     
  c3146b3524e97475c094343e848a7370.ttf  33.3 KiB          [emitted]                     
c52405129ae48097815d4da606003470.woff2  13.3 KiB          [emitted]                     
 ee26f57bc9e3619b022b2c0ea0fc4660.woff  87.7 KiB          [emitted]                     
Entrypoint main [big] = attendee.af3e7b864eca747edcc0.js
 [13] ./src/context/getChimeContext.ts 406 bytes {0} [built]
 [29] ./src/constants/routes.ts 205 bytes {0} [built]
 [38] ./src/providers/NotificationProvider/index.tsx 1.34 KiB {0} [built]
 [49] ./src/providers/LiveEventProvider.tsx 2.73 KiB {0} [built]
 [79] ./src/providers/VerifiedParticipantProvider.tsx 2.54 KiB {0} [built]
[130] ./src/providers/LiveEventMessagesProvider/index.tsx 12 KiB {0} [built]
[270] ./src/providers/FeatureProvider.tsx 1.43 KiB {0} [built]
[274] ./src/providers/StreamProvider.tsx 3.66 KiB {0} [built]
[317] ./src/apps/attendee/index.tsx 1.04 KiB {0} [built]
[318] ./src/app.global.css 576 bytes {0} [built]
[319] ./node_modules/css-loader/dist/cjs.js??ref--5-1!./src/app.global.css 6.86 KiB {0} [built]
[350] ./src/providers/ChimeProvider.tsx 29.7 KiB {0} [built]
[385] ./src/providers/CredentialsProvider.tsx 6.05 KiB {0} [built]
[388] ./src/providers/I18nProvider.tsx 790 bytes {0} [built]
[390] ./src/providers/LiveEventParticipantProvider.tsx 1.35 KiB {0} [built]
    + 543 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  3c45c98f6f6c1b009c082510d0255ef4.svg (699 KiB)
  18df003f90a7f1313b98b23384ec4f1b.svg (876 KiB)
  attendee.af3e7b864eca747edcc0.js (1.98 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (1.98 MiB)
      attendee.af3e7b864eca747edcc0.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child HtmlWebpackCompiler:
     1 asset
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [0] ./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html 1.15 KiB {0} [built]

> chime-sdk-react-demo@ package:moderator /Users/nidobda/Documents/dev/aws/amazon-chime-live-events
> webpack --config src/apps/moderator/webpack.prod.js

Shared dirname is /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/apps
Template: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/index.html
Prod path: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/dist
Hash: 112c108f6d5e70e9107e
Version: webpack 4.42.1
Time: 8872ms
Built at: 07/24/2020 1:33:17 AM
                                 Asset      Size  Chunks                                Chunk Names
  0747be4cba2f8600544f4f09645d971a.eot  33.6 KiB          [emitted]                     
  18df003f90a7f1313b98b23384ec4f1b.svg   876 KiB          [emitted]              [big]  
31015e861aeb30ff8355fab3e876a6bd.woff2  74.8 KiB          [emitted]                     
  32267f3b4db33a7c85ee31d9b317c9e4.ttf   198 KiB          [emitted]                     
  3c45c98f6f6c1b009c082510d0255ef4.svg   699 KiB          [emitted]              [big]  
  5818bbd07dfd7095920a592049f5d9bd.ttf   130 KiB          [emitted]                     
  607a77859353c0cc1882b51e1e269d38.eot   130 KiB          [emitted]                     
  7eefe04dc985dd4ad38bd7f4ec0c20a0.eot   198 KiB          [emitted]                     
 8b892b7904a39e37c53dc5a7534bb60e.woff  16.4 KiB          [emitted]                     
9451d5fee89b51a20f8a44c56a4f6c1a.woff2  77.6 KiB          [emitted]                     
 a940d584750708f5435ce2c523498ddb.woff   101 KiB          [emitted]                     
  ab47faeab8ddc5f809ec2a5377c1fbe4.svg   141 KiB          [emitted]                     
  c3146b3524e97475c094343e848a7370.ttf  33.3 KiB          [emitted]                     
c52405129ae48097815d4da606003470.woff2  13.3 KiB          [emitted]                     
 ee26f57bc9e3619b022b2c0ea0fc4660.woff  87.7 KiB          [emitted]                     
     moderator.7cd8532561959e3a2a95.js  1.49 MiB       0  [emitted] [immutable]  [big]  main
                        moderator.html  1.51 KiB          [emitted]                     
Entrypoint main [big] = moderator.7cd8532561959e3a2a95.js
  [4] ./src/hooks/useTranslate.tsx 419 bytes {0} [built]
 [14] ./src/providers/MetaStateProvider/index.tsx 1.22 KiB {0} [built]
 [17] ./src/providers/NotificationProvider/index.tsx 1.34 KiB {0} [built]
 [26] ./src/constants/routes.ts 205 bytes {0} [built]
 [36] ./node_modules/react-router-dom/esm/react-router-dom.js + 9 modules 70.8 KiB {0} [built]
      |    10 modules
 [38] ./src/providers/LiveEventProvider.tsx 2.73 KiB {0} [built]
 [45] ./src/providers/FeatureProvider.tsx 1.43 KiB {0} [built]
 [62] ./src/providers/VerifiedParticipantProvider.tsx 2.54 KiB {0} [built]
[152] ./src/providers/ChimeProvider.tsx 29.7 KiB {0} [built]
[307] ./src/apps/moderator/index.tsx 1.04 KiB {0} [built]
[308] ./src/app.global.css 576 bytes {0} [built]
[309] ./node_modules/css-loader/dist/cjs.js??ref--5-1!./src/app.global.css 6.86 KiB {0} [built]
[340] ./src/providers/CredentialsProvider.tsx 6.05 KiB {0} [built]
[378] ./src/providers/I18nProvider.tsx 790 bytes {0} [built]
[381] ./src/providers/LiveEventParticipantProvider.tsx 1.35 KiB {0} [built]
    + 552 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  3c45c98f6f6c1b009c082510d0255ef4.svg (699 KiB)
  18df003f90a7f1313b98b23384ec4f1b.svg (876 KiB)
  moderator.7cd8532561959e3a2a95.js (1.49 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (1.49 MiB)
      moderator.7cd8532561959e3a2a95.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child HtmlWebpackCompiler:
     1 asset
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [0] ./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html 1.15 KiB {0} [built]

> chime-sdk-react-demo@ package:talent /Users/nidobda/Documents/dev/aws/amazon-chime-live-events
> webpack --config src/apps/talent/webpack.prod.js

Shared dirname is /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/apps
Template: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/index.html
Prod path: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/dist
Hash: df71f2433f636193553a
Version: webpack 4.42.1
Time: 8345ms
Built at: 07/24/2020 1:33:27 AM
                                 Asset      Size  Chunks                                Chunk Names
  0747be4cba2f8600544f4f09645d971a.eot  33.6 KiB          [emitted]                     
  18df003f90a7f1313b98b23384ec4f1b.svg   876 KiB          [emitted]              [big]  
31015e861aeb30ff8355fab3e876a6bd.woff2  74.8 KiB          [emitted]                     
  32267f3b4db33a7c85ee31d9b317c9e4.ttf   198 KiB          [emitted]                     
  3c45c98f6f6c1b009c082510d0255ef4.svg   699 KiB          [emitted]              [big]  
  5818bbd07dfd7095920a592049f5d9bd.ttf   130 KiB          [emitted]                     
  607a77859353c0cc1882b51e1e269d38.eot   130 KiB          [emitted]                     
  7eefe04dc985dd4ad38bd7f4ec0c20a0.eot   198 KiB          [emitted]                     
 8b892b7904a39e37c53dc5a7534bb60e.woff  16.4 KiB          [emitted]                     
9451d5fee89b51a20f8a44c56a4f6c1a.woff2  77.6 KiB          [emitted]                     
 a940d584750708f5435ce2c523498ddb.woff   101 KiB          [emitted]                     
  ab47faeab8ddc5f809ec2a5377c1fbe4.svg   141 KiB          [emitted]                     
  c3146b3524e97475c094343e848a7370.ttf  33.3 KiB          [emitted]                     
c52405129ae48097815d4da606003470.woff2  13.3 KiB          [emitted]                     
 ee26f57bc9e3619b022b2c0ea0fc4660.woff  87.7 KiB          [emitted]                     
        talent.8d7c8eaeb56d4ed758a6.js   1.4 MiB       0  [emitted] [immutable]  [big]  main
                           talent.html  1.51 KiB          [emitted]                     
Entrypoint main [big] = talent.8d7c8eaeb56d4ed758a6.js
  [7] ./src/context/getChimeContext.ts 406 bytes {0} [built]
  [8] ./src/hooks/useTranslate.tsx 419 bytes {0} [built]
 [19] ./node_modules/react-router-dom/esm/react-router-dom.js + 9 modules 70.8 KiB {0} [built]
      |    10 modules
 [21] ./src/constants/routes.ts 205 bytes {0} [built]
 [24] ./src/context/getCredentialsContext.ts 1.02 KiB {0} [built]
 [49] ./src/providers/LiveEventProvider.tsx 2.73 KiB {0} [built]
 [64] ./src/providers/VerifiedParticipantProvider.tsx 2.54 KiB {0} [built]
[246] ./src/providers/FeatureProvider.tsx 1.43 KiB {0} [built]
[278] ./src/apps/talent/index.tsx 1.04 KiB {0} [built]
[279] ./src/app.global.css 576 bytes {0} [built]
[280] ./node_modules/css-loader/dist/cjs.js??ref--5-1!./src/app.global.css 6.86 KiB {0} [built]
[312] ./src/providers/ChimeProvider.tsx 29.7 KiB {0} [built]
[347] ./src/providers/I18nProvider.tsx 790 bytes {0} [built]
[349] ./src/providers/LiveEventParticipantProvider.tsx 1.35 KiB {0} [built]
[352] ./src/providers/CredentialsProvider.tsx 6.05 KiB {0} [built]
    + 461 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  3c45c98f6f6c1b009c082510d0255ef4.svg (699 KiB)
  18df003f90a7f1313b98b23384ec4f1b.svg (876 KiB)
  talent.8d7c8eaeb56d4ed758a6.js (1.4 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (1.4 MiB)
      talent.8d7c8eaeb56d4ed758a6.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child HtmlWebpackCompiler:
     1 asset
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [0] ./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html 1.15 KiB {0} [built]

> chime-sdk-react-demo@ package:broadcast /Users/nidobda/Documents/dev/aws/amazon-chime-live-events
> webpack --config src/apps/broadcast/webpack.prod.js

Shared dirname is /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/apps
Template: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/src/index.html
Prod path: /Users/nidobda/Documents/dev/aws/amazon-chime-live-events/dist
Hash: f710b874fb2d1da81338
Version: webpack 4.42.1
Time: 8173ms
Built at: 07/24/2020 1:33:36 AM
                                 Asset      Size  Chunks                                Chunk Names
  0747be4cba2f8600544f4f09645d971a.eot  33.6 KiB          [emitted]                     
  18df003f90a7f1313b98b23384ec4f1b.svg   876 KiB          [emitted]              [big]  
31015e861aeb30ff8355fab3e876a6bd.woff2  74.8 KiB          [emitted]                     
  32267f3b4db33a7c85ee31d9b317c9e4.ttf   198 KiB          [emitted]                     
  3c45c98f6f6c1b009c082510d0255ef4.svg   699 KiB          [emitted]              [big]  
  5818bbd07dfd7095920a592049f5d9bd.ttf   130 KiB          [emitted]                     
  607a77859353c0cc1882b51e1e269d38.eot   130 KiB          [emitted]                     
  7eefe04dc985dd4ad38bd7f4ec0c20a0.eot   198 KiB          [emitted]                     
 8b892b7904a39e37c53dc5a7534bb60e.woff  16.4 KiB          [emitted]                     
9451d5fee89b51a20f8a44c56a4f6c1a.woff2  77.6 KiB          [emitted]                     
 a940d584750708f5435ce2c523498ddb.woff   101 KiB          [emitted]                     
  ab47faeab8ddc5f809ec2a5377c1fbe4.svg   141 KiB          [emitted]                     
     broadcast.8b238604782dec90285c.js  1.35 MiB       0  [emitted] [immutable]  [big]  main
                        broadcast.html  1.51 KiB          [emitted]                     
  c3146b3524e97475c094343e848a7370.ttf  33.3 KiB          [emitted]                     
c52405129ae48097815d4da606003470.woff2  13.3 KiB          [emitted]                     
 ee26f57bc9e3619b022b2c0ea0fc4660.woff  87.7 KiB          [emitted]                     
Entrypoint main [big] = broadcast.8b238604782dec90285c.js
  [5] ./src/context/getChimeContext.ts 406 bytes {0} [built]
 [10] ./src/hooks/useTranslate.tsx 419 bytes {0} [built]
 [30] ./src/providers/LiveAttendeesProvider.tsx 4.63 KiB {0} [built]
 [36] ./node_modules/react-router-dom/esm/react-router-dom.js + 9 modules 70.8 KiB {0} [built]
      |    10 modules
 [45] ./src/providers/LocalTileProvider.tsx 6.2 KiB {0} [built]
 [60] ./src/providers/VerifiedParticipantProvider.tsx 2.54 KiB {0} [built]
[108] ./src/providers/LiveEventProvider.tsx 2.73 KiB {0} [built]
[274] ./src/apps/broadcast/index.tsx 1.04 KiB {0} [built]
[275] ./src/app.global.css 576 bytes {0} [built]
[276] ./node_modules/css-loader/dist/cjs.js??ref--5-1!./src/app.global.css 6.86 KiB {0} [built]
[308] ./src/providers/ChimeProvider.tsx 29.7 KiB {0} [built]
[343] ./src/providers/I18nProvider.tsx 790 bytes {0} [built]
[345] ./src/providers/LiveEventParticipantProvider.tsx 1.35 KiB {0} [built]
[348] ./src/providers/CredentialsProvider.tsx 6.05 KiB {0} [built]
[350] ./src/providers/FeatureProvider.tsx 1.43 KiB {0} [built]
    + 431 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  3c45c98f6f6c1b009c082510d0255ef4.svg (699 KiB)
  18df003f90a7f1313b98b23384ec4f1b.svg (876 KiB)
  broadcast.8b238604782dec90285c.js (1.35 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (1.35 MiB)
      broadcast.8b238604782dec90285c.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child HtmlWebpackCompiler:
     1 asset
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [0] ./node_modules/html-webpack-plugin/lib/loader.js!./src/index.html 1.15 KiB {0} [built]
,
Using region us-east-1, bucket mdliveversion4, stack mdsliveversion4, domain undefined ACM cert arn undefined
Creating deployment bucket
Creating S3 bucket mdliveversion4
,aws-cli/2.0.31 Python/3.7.4 Darwin/19.5.0 botocore/2.0.0dev35

SAM CLI, version 0.53.0

Using region us-east-1, bucket mdliveversion4-broadcast, stack mdsliveversion4-broadcast
Creating S3 bucket mdliveversion4-broadcast
Creating ECR repository with name: mdsliveversion4-broadcast
The ECR repository is created, reposiroty URI is 905351406809.dkr.ecr.us-east-1.amazonaws.com/mdsliveversion4-broadcast
Building Docker image and push it into ECR repository: 905351406809.dkr.ecr.us-east-1.amazonaws.com/mdsliveversion4-broadcast...
Valid ECR URI format
aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 905351406809.dkr.ecr.us-east-1.amazonaws.com/mdsliveversion4-broadcast
Login Succeeded
docker build -t mdsliveversion4-broadcast .
Deploying Broadcast API application

	Deploying with following values
	===============================
	Stack name                 : mdsliveversion4-broadcast
	Region                     : us-east-1
	Confirm changeset          : False
	Deployment s3 bucket       : mdliveversion4-broadcast
	Capabilities               : ["CAPABILITY_IAM", "CAPABILITY_AUTO_EXPAND"]
	Parameter overrides        : {'ECRDockerImageArn': '905351406809.dkr.ecr.us-east-1.amazonaws.com/mdsliveversion4-broadcast:latest'}

Initiating deployment
=====================

Waiting for changeset to be created..

CloudFormation stack changeset
------------------------------------------------------------------------------------------------
Operation                        LogicalResourceId                ResourceType                   
------------------------------------------------------------------------------------------------
+ Add                            BroadcastECSLogGroup             AWS::Logs::LogGroup            
+ Add                            BroadcastLambdaExecutionRole     AWS::IAM::Role                 
+ Add                            BroadcastLambdaFunction          AWS::Lambda::Function          
+ Add                            ContainerInstances               AWS::AutoScaling::LaunchConfig 
                                                                  uration                        
+ Add                            DefaultPublicRoute               AWS::EC2::Route                
+ Add                            EC2InstanceProfile               AWS::IAM::InstanceProfile      
+ Add                            EC2Role                          AWS::IAM::Role                 
+ Add                            ECSAutoScalingGroup              AWS::AutoScaling::AutoScalingG 
                                                                  roup                           
+ Add                            ECSBroadcastTaskDefinition       AWS::ECS::TaskDefinition       
+ Add                            ECSCluster                       AWS::ECS::Cluster              
+ Add                            EcsSecurityGroupALBports         AWS::EC2::SecurityGroupIngress 
+ Add                            EcsSecurityGroupHTTPinbound      AWS::EC2::SecurityGroupIngress 
+ Add                            EcsSecurityGroupSSHinbound       AWS::EC2::SecurityGroupIngress 
+ Add                            EcsSecurityGroup                 AWS::EC2::SecurityGroup        
+ Add                            InternetGatewayAttachment        AWS::EC2::VPCGatewayAttachment 
+ Add                            InternetGateway                  AWS::EC2::InternetGateway      
+ Add                            PublicRouteTable                 AWS::EC2::RouteTable           
+ Add                            PublicSubnet1RouteTableAssocia   AWS::EC2::SubnetRouteTableAsso 
                                 tion                             ciation                        
+ Add                            PublicSubnet1                    AWS::EC2::Subnet               
+ Add                            PublicSubnet2RouteTableAssocia   AWS::EC2::SubnetRouteTableAsso 
                                 tion                             ciation                        
+ Add                            PublicSubnet2                    AWS::EC2::Subnet               
+ Add                            VPC                              AWS::EC2::VPC                  
------------------------------------------------------------------------------------------------

Changeset created successfully. arn:aws:cloudformation:us-east-1:905351406809:changeSet/samcli-deploy1595536453/e99f756b-0aa7-472f-bec6-d2ed30cfe419


2020-07-24 01:34:30 - Waiting for stack create/update to complete

CloudFormation events from changeset
-------------------------------------------------------------------------------------------------
ResourceStatus           ResourceType             LogicalResourceId        ResourceStatusReason   
-------------------------------------------------------------------------------------------------
CREATE_IN_PROGRESS       AWS::EC2::VPC            VPC                      -                      
CREATE_IN_PROGRESS       AWS::IAM::Role           EC2Role                  -                      
CREATE_IN_PROGRESS       AWS::EC2::InternetGate   InternetGateway          -                      
                         way                                                                      
CREATE_IN_PROGRESS       AWS::Logs::LogGroup      BroadcastECSLogGroup     -                      
CREATE_IN_PROGRESS       AWS::IAM::Role           BroadcastLambdaExecuti   -                      
                                                  onRole                                          
CREATE_IN_PROGRESS       AWS::ECS::Cluster        ECSCluster               -                      
CREATE_FAILED            AWS::IAM::Role           EC2Role                  Resource creation      
                                                                           cancelled              
CREATE_FAILED            AWS::EC2::VPC            VPC                      Resource creation      
                                                                           cancelled              
CREATE_FAILED            AWS::EC2::InternetGate   InternetGateway          Resource creation      
                         way                                               cancelled              
CREATE_FAILED            AWS::IAM::Role           BroadcastLambdaExecuti   Resource creation      
                                                  onRole                   cancelled              
CREATE_IN_PROGRESS       AWS::IAM::Role           EC2Role                  Resource creation      
                                                                           Initiated              
CREATE_IN_PROGRESS       AWS::EC2::VPC            VPC                      Resource creation      
                                                                           Initiated              
CREATE_IN_PROGRESS       AWS::EC2::InternetGate   InternetGateway          Resource creation      
                         way                                               Initiated              
CREATE_FAILED            AWS::Logs::LogGroup      BroadcastECSLogGroup     log-group-ecs-         
                                                                           broadcast-lambda       
                                                                           already exists in      
                                                                           stack arn:aws:cloudfor 
                                                                           mation:us-east-1:90535 
                                                                           1406809:stack/mdslivev 
                                                                           ersion2-broadcast/c639 
                                                                           4130-cd20-11ea-        
                                                                           bc6a-0aa958b6917a      
CREATE_IN_PROGRESS       AWS::IAM::Role           BroadcastLambdaExecuti   Resource creation      
                                                  onRole                   Initiated              
CREATE_FAILED            AWS::ECS::Cluster        ECSCluster               ec2-cluster-broadcast- 
                                                                           lambda already exists  
                                                                           in stack arn:aws:cloud 
                                                                           formation:us-east-1:90 
                                                                           5351406809:stack/mdsli 
                                                                           veversion2-broadcast/c 
                                                                           6394130-cd20-11ea-     
                                                                           bc6a-0aa958b6917a      
ROLLBACK_IN_PROGRESS     AWS::CloudFormation::S   mdsliveversion4-broadc   The following          
                         tack                     ast                      resource(s) failed to  
                                                                           create: [BroadcastLamb 
                                                                           daExecutionRole,       
                                                                           InternetGateway, VPC,  
                                                                           EC2Role,               
                                                                           BroadcastECSLogGroup,  
                                                                           ECSCluster]. .         
                                                                           Rollback requested by  
                                                                           user.                  
DELETE_IN_PROGRESS       AWS::EC2::VPC            VPC                      -                      
DELETE_IN_PROGRESS       AWS::IAM::Role           EC2Role                  -                      
DELETE_COMPLETE          AWS::ECS::Cluster        ECSCluster               -                      
DELETE_COMPLETE          AWS::Logs::LogGroup      BroadcastECSLogGroup     -                      
DELETE_IN_PROGRESS       AWS::IAM::Role           BroadcastLambdaExecuti   -                      
                                                  onRole                                          
DELETE_IN_PROGRESS       AWS::EC2::InternetGate   InternetGateway          -                      
                         way                                                                      
DELETE_COMPLETE          AWS::IAM::Role           EC2Role                  -                      
DELETE_COMPLETE          AWS::IAM::Role           BroadcastLambdaExecuti   -                      
                                                  onRole                                          
DELETE_COMPLETE          AWS::EC2::VPC            VPC                      -                      
DELETE_COMPLETE          AWS::EC2::InternetGate   InternetGateway          -                      
                         way                                                                      
ROLLBACK_COMPLETE        AWS::CloudFormation::S   mdsliveversion4-broadc   -                      
                         tack                     ast                                             
-------------------------------------------------------------------------------------------------

Command sam failed with exit code 1 signal null
Uploading to 954276bde540ef29a7b5ddabc1f03673  1163 / 1163.0  (100.00%)
Uploading to 4e2a08cf288b352566d0977250fe0cc6.template  10173 / 10173.0  (100.00%)
Error: Failed to create/update the stack: mdsliveversion4-broadcast, Waiter StackCreateComplete failed: Waiter encountered a terminal failure state

,Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
make: *** [build_upload] Error 1

Command node failed with exit code 1 signal null
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
make: *** [build_upload] Error 1

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,2 @@
+node_modules
+dist/

--- a/transcoding/template.yaml
+++ b/transcoding/template.yaml
@@ -12,7 +12,7 @@ Parameters:
     Default: 8192
   ECSContainerName:
     Type: String
-    Default: 'container-broadcast-lambda'
+    Default: container-
   ECSContainerCpu:
     Type: Number
     Default: 4096
@@ -25,7 +25,7 @@ Parameters:
   ECSClusterName:
     Type: String
     Description: 'Specifies the ECS Cluster Name with which the resources would be associated'
-    Default: 'ec2-cluster-broadcast-lambda'
+    Default: ec2-cluster-
   ECRDockerImageArn:
     Type: String
     Description: 'ARN of the docker image stored in ECR along with the tag'
@@ -54,7 +54,7 @@ Parameters:
   EnvironmentName:
     Description: An environment name that is prefixed to resource names
     Type: String
-    Default: environment-broadcast-lambda
+    Default: environment-
   VpcCIDR:
     Description: Please enter the IP range (CIDR notation) for this VPC
     Type: String
@@ -85,13 +85,13 @@ Resources:
   BroadcastECSLogGroup:
     Type: AWS::Logs::LogGroup
     Properties:
-      LogGroupName: log-group-ecs-broadcast-lambda
+      LogGroupName: !Sub 'log-group-ecs-${AWS::StackName}-lambda'
       RetentionInDays: 365
   
   BroadcastLambdaFunction:
     Type: 'AWS::Serverless::Function'
     Properties:
-      FunctionName: broadcast-lambda
+      FunctionName: !Sub '${AWS::StackName}-lambda'
       Description: Lambda to interact with ECS for starting and stopping broadcast.
       Handler: index.handler
       Role: 
@@ -103,8 +103,7 @@ Resources:
         Variables:
           ecsClusterArn:
             Fn::GetAtt: [ECSCluster, Arn]
-          ecsContainerName: 
-            Ref: ECSContainerName
+          ecsContainerName: !Sub '${ECSContainerName}-${AWS::StackName}'
           ecsTaskDefinationArn: 
             Ref: ECSBroadcastTaskDefinition
       CodeUri: src/
@@ -123,8 +122,7 @@ Resources:
               Host:
                 SourcePath: '/run/dbus/system_bus_socket:/run/dbus/system_bus_socket'
         ContainerDefinitions:
-            - Name: 
-                Ref: ECSContainerName
+            - Name: !Sub '${ECSContainerName}-${AWS::StackName}'
               Cpu: 
                 Ref: ECSContainerCpu
               Memory: 
@@ -141,16 +139,14 @@ Resources:
                     Ref: BroadcastECSLogGroup
                   awslogs-region:
                     Ref: AWS::Region
-                  awslogs-stream-prefix:
-                    Ref: ECSContainerName
+                  awslogs-stream-prefix: !Sub '${ECSContainerName}-${AWS::StackName}'
               LinuxParameters:
                 SharedMemorySize: 2048
 
   ECSCluster:
     Type: 'AWS::ECS::Cluster'
     Properties:
-        ClusterName: 
-            Ref: ECSClusterName
+        ClusterName: !Sub '${ECSClusterName}-${AWS::StackName}'
 # Refer: https://docs.aws.amazon.com/codebuild/latest/userguide/cloudformation-vpc-template.html
   
   VPC:
@@ -162,13 +158,13 @@ Resources:
       InstanceTenancy: default
       Tags:
         - Key: Name
-          Value: !Ref EnvironmentName
+          Value: !Sub '${EnvironmentName}-${AWS::StackName}'
   InternetGateway:
     Type: AWS::EC2::InternetGateway
     Properties:
       Tags:
         - Key: Name
-          Value: !Ref EnvironmentName
+          Value:  !Sub '${EnvironmentName}-${AWS::StackName}'
   InternetGatewayAttachment:
     Type: AWS::EC2::VPCGatewayAttachment
     Properties:
@@ -183,7 +179,7 @@ Resources:
       MapPublicIpOnLaunch: true
       Tags:
         - Key: Name
-          Value: !Sub ${EnvironmentName} Public Subnet (AZ1)
+          Value: !Sub ${EnvironmentName}-${AWS::StackName} Public Subnet (AZ1)
   PublicSubnet2:
     Type: AWS::EC2::Subnet
     Properties:
@@ -193,14 +189,14 @@ Resources:
       MapPublicIpOnLaunch: true
       Tags:
         - Key: Name
-          Value: !Sub ${EnvironmentName} Public Subnet 
+          Value: !Sub ${EnvironmentName}-${AWS::StackName} Public Subnet 
   PublicRouteTable:
     Type: AWS::EC2::RouteTable
     Properties:
       VpcId: !Ref VPC
       Tags:
         - Key: Name
-          Value: !Sub ${EnvironmentName} Public Routes
+          Value: !Sub ${EnvironmentName}-${AWS::StackName} Public Routes
   DefaultPublicRoute:
     Type: AWS::EC2::Route
     DependsOn: InternetGatewayAttachment

